### DISCUSSION TITLE
Compatibility of OpenDDSharp with older versions of OpenDDS

### DISCUSSION THREAD

**Discussion by RinoGuideDog** [OpenDDS/OpenDDS#4865] [Q&A] [answered]
is it possible for example to set up open dds sharp to be using open OpenDDS version 3.18.0 with ACE+TAO 2.2a-p21?

**Comment by jmmorato** [comment, marked as answer]
I think some breaking changes in the API would need to be adapted but if I'm not wrong version 3.18 and latest version of OpenDDS are compatible on the wire, why do you need that specific version?

**Reply by RinoGuideDog** [reply]
having to deal with a app thats a pain in the arse to be honest ty for the help
